### PR TITLE
move "gowebcam" enabled styles over to css

### PR DESCRIPTION
**Pull request by jcalado**

### CODE REVIEW

**Review by steveseguin — APPROVED** [review]

File: .github/ci-generateTranslations.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  storageQuota: 10000000,
+  pretendToBeVisual: false,
+}).window.document;
+
+function downloadTranslation(filename, trans = {}) {
+  // downloads the current translation to a file
+  console.log("Downloading translation: " + filename);
+  const textDoc = JSON.stringify(trans, null, 2);
+
+  fs.writeFile(`translations/${filename}.json`, textDoc, function (err) {
```
Comment (filiptronicek) [inline .github/ci-generateTranslations.js:28]:
```suggestion
  fs.writeFile(`translations/${filename}.json`, textDoc, (err) => {
```

File: .github/ci-generateTranslations.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  // updates the website with a specific translation
+  let data = await axios({
+    method: "get",
+    url: `https://raw.githubusercontent.com/steveseguin/obsninja/master/translations/${filename}.json?${(
+      Math.random() * 100
+    ).toString()}`,
+  })
+    .then(function (response) {
+      return response.data;
+    })
```
Comment (filiptronicek) [inline .github/ci-generateTranslations.js:47]:
```suggestion
    .then((response) => response.data)
```

File: .github/ci-generateTranslations.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    .catch(function (error) {
```
Comment (filiptronicek) [inline .github/ci-generateTranslations.js:48]:
```suggestion
    .catch((error) => {
```

File: .github/ci-generateTranslations.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+combinedTrans.innerHTML = defaultTrans;
+combinedTrans.placeholders = defaultTransPlaceholders;
+
+var counter = 0;
+for (var i = 0; i < updateList.length; i++) {
+  const lang = updateList[i];
+
+  var translation = updateTranslation(lang); // we don't need to worry about DATA.
+  updateTranslation(lang)
+    .then(function (translation) {
```
Comment (filiptronicek) [inline .github/ci-generateTranslations.js:136]:
```suggestion
    .then((translation) => {
```

File: .github/ci-generateTranslations.js [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+      });
+
+      // //// DOWNLOAD UPDATED TRANSLATION
+      const outputTrans = {};
+      outputTrans["titles"] = newTransTitles;
+      outputTrans["innerHTML"] = newTrans;
+      outputTrans["placeholders"] = newPlaceholders;
+      downloadTranslation(lang, outputTrans);
+    })
+    .catch(function (error) {
```
Comment (filiptronicek) [inline .github/ci-generateTranslations.js:165]:
```suggestion
    .catch((error) => {
```

**Review by filiptronicek — CHANGES_REQUESTED** [review]
All good, just a couple code style suggestions

**Comment by filiptronicek** [comment]
Oh, I guess I was too late 😅

**Comment by steveseguin** [comment]
It's never too late.

**Pull request merged**